Makefile

```diff
@@ -185,7 +189,7 @@ $(if $(wildcard THRIFT_FILES),,$(error idls/ submodule must exist, or build will
 endef
 
 # codegen is done when thrift is done (it's just a naming-convenience, $(BUILD)/thrift would be fine too)
-$(BUILD)/codegen: $(BUILD)/thrift | $(BUILD)
+$(BUILD)/codegen: | $(BUILD)
```

Comment:
just unnecessary since it's defined above, and re-definitions merge dependencies